doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -2598,12 +2674,16 @@ \subsubsection{Field: External Shading Fraction Schedule Name}\label{field-exter
 
 \subsubsection{Field: Surrounding Surfaces Object Name}\label{field-surrounding-surfaces-object-name}
 
-This field is used to import surrounding surfaces properties (temperature and view factors to the external surface) from external calculation. This should be a name of a \textbf{\hyperref[surfacePropertysurroundingSurfaces]{SurfaceProperty:SurroundingSurfaces}} object.
+This field is used to import surrounding surfaces properties (temperature and view factors to the external surface) from external calculation. This should be a name of a \textbf{\hyperref[surfacePropertySurroundingSurfaces]{SurfaceProperty:SurroundingSurfaces}} object.
 
 \subsubsection{Field: Outdoor Air Node Name}\label{field-outdoor-air-node-name}
 
 This field is used to import local environmental data from local outdoor air nodes, including dry and wet bulb temperature, wind speed and wind direction. This should be a name of a \textbf{\hyperref[outdoorairnode]{OutdoorAir:Node}} object.
 
+\subsubsection{Field: Ground Surfaces Object Name}\label{field-ground-surfaces-object-name}
+
+This field is used to import ground surfaces properties (ground surface temperature, solar reflectance and view factors to the external surface) from external calculation. This should be a name of a \textbf{\hyperref[surfacePropertyGroundSurfaces]{SurfaceProperty:GroundSurfaces}} object.
+
```

Comment:
New field added to `SurfaceProperty:LocalEnvironment` object. The new field is used to enter the name of the `SurfaceProperty:GroundSurfaces` object.